apps/docs/src/app/core/component-docs/message-toast/message-toast-header/message-toast-header.component.html

```diff
@@ -0,0 +1,25 @@
+<header>Message Toast</header>
+<description
+    >Message Toast is a small, non-disruptive popup for success messages that disappears automatically after a few
+    seconds. <br /><br />
+
+    <b>Usage</b><br />
+    Use Message Toast in the following cases:<br />
+    - To display a short success message. <br />
+    - To avoid interrupting the user while they are performing an action.<br />
+    - To confirm a successful action.<br /><br />
```

Comment:
Use message toast if:
- You want to display a short success message.
- You want to avoid interrupting the user while they are performing an action.
- You want the users to have confirmation for their successful actions.

Do not use the message toast if:
- You want to display an error or warning message.
- You want to interrupt users while they are performing an action.
- You want to make sure that users read the message before they leave the page.